tests/unhappypath/unhappypath_test.go

```diff
@@ -0,0 +1,281 @@
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/pkg/kvdbutils"
+	"github.com/portworx/torpedo/pkg/snapshotutils"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	storkops = stork.Instance()
+)
+
+const (
+	dropPercentage      = 20
+	delayInMilliseconds = 250
+	//24 hours
+	totalTimeInHours              = 24
+	errorPersistTimeInMinutes     = 60 * time.Minute
+	snapshotScheduleRetryInterval = 10 * time.Second
+	snapshotScheduleRetryTimeout  = 5 * time.Minute
+)
+
+func TestBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Basic", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+	if Inst().N.IsUsingSSH() {
+		for _, anode := range node.GetWorkerNodes() {
+			// TODO need to remove tcpdump installation using yum
+			// once https://portworx.atlassian.net/browse/PTX-12174 is resolved.
+			logrus.Infof("installing tcpdump on node %s", anode.Name)
+			cmd := "yum install -y tcpdump"
+			_, err := Inst().N.RunCommandWithNoRetry(anode, cmd, node.ConnectionOpts{
+				Timeout:         1 * time.Minute,
+				TimeBeforeRetry: 5 * time.Second,
+				Sudo:            true,
+			})
+			if err != nil {
+				logrus.Warnf("failed to install tcpdump on node %s: %v", anode.Name, err)
+				break
+			}
+			//Check tcpdump command
+			cmd = "compgen -c | grep tcpdump"
+			out, errInSSH := Inst().N.RunCommandWithNoRetry(anode, cmd, node.ConnectionOpts{
+				Timeout:         1 * time.Minute,
+				TimeBeforeRetry: 5 * time.Second,
+				Sudo:            true,
+			})
+			if err != nil {
+				logrus.Warnf("failed to install tcpdump on node %s: %v", anode.Name, errInSSH)
+			}
+			if !strings.Contains(out, "tcpdump") {
+				logrus.Warnf("TCP dump is not avaiable in %s:", anode.Name)
+			}
+		}
+	}
+})
+
+// This test is to verify stability of the system when there  is  a network error on the system.
+var _ = Describe("{NetworkErrorInjection}", func() {
+	var testrailID = 3526435
+	injectionType := "drop"
+	//TODO need to fix this issue later.
+	// testrailID corresponds to: https://portworx.testrail.net/index.php?/cases/view/35264
+	var runID int
+	JustBeforeEach(func() {
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	It("Inject network error while applications are running", func() {
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			logrus.Infof("Iteration number %d", i)
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("applicationscaleup-%d", i))...)
+		}
+		currentTime := time.Now()
+		timeToExecuteTest := time.Now().Local().Add(time.Hour*time.Duration(totalTimeInHours) +
+			time.Minute*time.Duration(0) +
```

Comment:
Curious as to why do we need to add Minutes and Seconds if they are set to `0` anyways.

Reply:
Removed minutes and  seconds, as test is running for 24 hours